go.mod

```diff
@@ -1,20 +1,22 @@
 module github.com/fullstorydev/grpchan
 
-go 1.18
+go 1.22.7
```

Comment:
q: is the the minimum version that can be supported now?  On other projects we've general run the latest 3 versions (e.g. 1.21, 1.22, 1.23).